config/cubedos_config.h

```diff
@@ -9,7 +9,7 @@
 
 #define ALIRE_HOST_ARCH "x86_64"
 
-#define ALIRE_HOST_DISTRO "ubuntu"
+#define ALIRE_HOST_DISTRO "arch"
```

Comment:
I did this on ym local machine instead of the VM- this file should be in git-ignore